chcemvediet/apps/inforequests/forms/obligee_action.py

```diff
@@ -403,6 +451,36 @@ def post_transition(self):
 
         return res
 
+class DisclosureLevelFork2(ObligeeActionStep):
```

Comment:
@martinmacko47 V grafe `obligee-action-wizard.svg` mame dva stavy `DL`, pricom robia dve rozne veci. Navrhujem `DisclosureLevelFork` premenovat na `ReversionLevelFork`. Obe budu moct aj nadalej pracovat s rovnakou premennou `self.wizard.values['disclosure_level']`.

Taktiez v dokumentacii skratku `DL` prepisat na `RL` a pridat ju do legendy.

Reply:
@viliambalaz  V oboch pripadoch to je `disclosure_level`, pretoze v oboch pripadoch to je hodnota pre `Action.disclosure_level`. Preto aj skratka je v oboch pripadoch `DL`.

Jednotlive kroky vo wizarde by som asi nazval `DisclosureLevelFork` a `ReversionDisclosureLevelFork`.